lib/kafo/exit_handler.rb

```diff
@@ -10,12 +10,13 @@ def initialize
 
     def error_codes
       @error_codes ||= {
-          :invalid_system => 20,
-          :invalid_values => 21,
-          :manifest_error => 22,
-          :no_answer_file => 23,
-          :unknown_module => 24,
-          :defaults_error => 25
+          :invalid_system   => 20,
```

Comment:
luckily this is not puppet, don't align hash rockets, btw @iNecas will be mad for breaking this :-) make sure you document this in [README](https://github.com/theforeman/kafo#exit-code)